tests/data/keys/privkey.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
+MHQCAQEEIAqhMWlmwcHwa2pXlyxUfPUvKMdrHHxGAkKz0EfHrlZpoAcGBSuBBAAK
+oUQDQgAEhJIpSysqJlsr0+nAwQDYaqk4hkLmU+2Pje5jCpI6QfakJD+bVrXqF+5Z
+xbwEh+e+lrhDLfj9+jJTOda4WD83Ng==
+-----END EC PRIVATE KEY-----
```

Comment:
Please document the command(s) to create the key in case somebody somewhen needs to update it for one reason or another. The commit message is fine for that.